linkatos/message.py

```diff
@@ -1,10 +1,10 @@
 import re
 
+
 def extract_url(message):
-    """
-    Returns the first url in a message. If there aren't any returns None
-    """
-    answer = re.search("(?:\s|^)<(https?://[\w./?+&+%$!#=\-_]+)>(?:\s|$)", message)
+    # Returns the first url in a message. If there aren't any returns None
+    url_re = "(?:\s|^)<(https?://[\w./?+&+%$!#=\-_]+)>(?:\s|$)"
+    answer = re.search(url_re, message)
```

Comment:
or actually, in this case I would use `captures` as in "group captures" because it is the name of what this variable holds.